src/components/TopBar.js

```diff
@@ -93,6 +79,48 @@ const betaTag = b({
   }
 }, 'BETA')
 
+const DropDownSubItem = props => {
+  return h(Clickable, {
+    as: 'a',
+    style: styles.nav.dropDownItem,
+    hover: { backgroundColor: colors.dark(0.55) },
+    ...props
+  })
+}
+
+const DropDownSection = props => {
+  const { titleIcon, title, isOpened, onClick, children } = props
+
+  return h(Fragment, [
+    h(Clickable, {
+      style: styles.nav.item,
+      hover: { backgroundColor: colors.dark(0.55) },
+      onClick
+    }, [
+      titleIcon && div({ style: styles.nav.icon }, [
+        icon(titleIcon, {
+          className: 'is-solid',
+          size: 24
+        })
+      ]),
+      div({
+        style: {
+          whiteSpace: 'nowrap',
+          overflow: 'hidden',
+          textOverflow: 'ellipsis'
+        }
+      }, [title]),
+      div({ style: { flexGrow: 1 } }),
+      icon(`angle ${isOpened ? 'up' : 'down'}`, //arrow isn't flipping
+        {
+          size: 18,
+          style: { flex: 'none' }
+        })
+    ]),
+    h(RCollapse, { isOpened }, [children])
```

Comment:
There's a warning in the console about `isOpened` being `undefined`. We should add a constructor and explicitly initialize the state to `false`.